### ISSUE TITLE
Measure Loop

### ISSUE THREAD

**Issue by db5050** [paulrosen/abcjs#714]
I use audio for play. Somehow, I couldn't make it ring between two measures. There is a setprogress function. The thing that might work for me here is the beat. However, this method will not give accurate results when there is more than one rhythm in the song. The other percent and millisecond options are already unsuitable for the note. Actually I want something simple. For example, I want to ring between measure 5 and measure 15. Can I do this with abcjs? (Assuming there are 5/8.4/4.7/8 rhythms in a song and lots of repeat marks)

**Comment by paulrosen** [comment]
I'm not sure what you mean by "ring". Do you mean playing a really long note? What instrument? There is a fixed length to the soundfont notes so it probably won't work to try to play a note more than a few seconds. For some instruments that don't have attack or decay the soundfont could be repeated but that doesn't happen yet.

**Comment by db5050** [comment]
![Ekran Alıntısı](https://user-images.githubusercontent.com/4678849/127735513-5a8cf039-bf8b-4b42-b0cd-d0463ba03775.PNG)
How can I ring between the 3rd and 5th measures?

**Comment by paulrosen** [comment]
Do you mean "play the `d` note for the whole 3rd, 4th, and 5th measures"? Sorry to not understand.

**Comment by db5050** [comment]
no. I don't know how to explain it any other way. It's not about playing notes, that's what I want. When I press the play button, all the 3rd measure, all the 4th measure, all the 5th measure will be played. Then it will return to the 3rd measure and this cycle will continue continuously. This feature is necessary so that students can continuously study the measure they have chosen.

**Comment by paulrosen** [comment]
Oh, I see - that is probably a bug. I'll look into it. That should work if there isn't a time sig change but I don't think I tested what you have.

**Comment by db5050** [comment]
Any progress on this issue? Actually, this will work for me. If I can see the "beat number" value of the note I clicked, I can repeat the parts I want.
